src/autobridgecalculator.js

```diff
@@ -70,14 +73,17 @@ AutobridgeCalculator.prototype.calculate = function() {
       continue;
     }
 
-    if (legOneTakerGetsFunded.compareTo(legTwoTakerPaysFunded) > 0) {
+    //  using private fields for speed
```

Comment:
This is here a really special cases, that shouldn't be used otherwise.
In compare, I'm sure that other object is `Amount` and comparable, so I don't want to create new object and run comparable checks on it.
As for creation, I want to bypass some checks too, because I'm reasonably sure that my data is valid, and also don't want to create more objects that necessary.